ClinGen/clincoded
# Connects to #1369. Connects to #1637. Connects to #1657. Connects to #1665. Connects to #1667. Connects to #1678.

**Steps to test #1369:**
1. In a Gene-Disease record, start a new **Experimental** curation and select the **Expression** type.
2. Fill out other required fields as needed.
3. In the **UBERON ID** input field, enter the ID with either colon or underscore (e.g. ":" or "_").
4. Expect to be able to save the form successfully.
5. Upon returning to the Gene-Disease record page, click on the **View/Score** link of the newly curated evidence in the **Experimental** curation palette.
6. On the view-only page, click on the UBERON ID linkout and expect to see a new window/tab consisting the external data associated with ID.
7. Start another new **Experimental** curation and select the **Functional Alteration** type. Repeat the testing steps above for the EFO and CL IDs. Expect to see same behaviors.
8. Start another new **Experimental** curation and select the **Model Systems** or **Rescue** type. Repeat the testing steps above for the EFO and CL IDs. Expect to see same behaviors.

**Steps to test #1637:**
1. In the VCI, expect to see the previously named "Predictors" tab being renamed to "Variant Type".
2. Cycle through different tabs by clicking and expect to see the tabs working without errors or stalling.

**Steps to test #1657:**
1. In a Gene-Disease record, curate some evidence and score them.
2. Go to the Classification Matrix and save it to proceed to the next page on which you can provision the saved classification.
3. Provision and approve the classification, and expect to see the snapshot panel with the title spelled correctly as **Saved Provisional and Approved Classification(s)**.

**Steps to test #1665:**
1. On the **Basic Info** tab in the VCI, expect to see the SCV panel title now with **(Germline SCVs only)** appended to **Interpretations Submitted to ClinVar**.

**Steps to test #1667:**
1. Start a new Gene-Disease record with the **Autosomal dominant inheritance** Mode of Inheritance.
2. Curate 2 new Proband evidence and score each of them with a modified score of **0.1**.
3. Go to the Classification Matrix and expect to see the **Total Points** for the curated evidence to be **0.2** rather than **0.188888...888889** precision number in the matrix.
4. Save the calculated points and proceed to the next page.
5. Expect to see the same **0.2** points in the matrix on the subsequent page.
